app/controllers/notes_controller.rb

```diff
@@ -0,0 +1,29 @@
+class NotesController < ApplicationController
+  def new
```

Comment:
use load and authorize, and add to ability 
do not allow everyone permission to add notes